packages/ledger-qldb/README.md

```diff
@@ -29,3 +29,16 @@ npm install --save @sidetree/qldb
 npm install
 npm run test
 ```
+
+## Is QLDB FIPS compliant ?
+
+- QLDB uses a Merkle Tree with the SHA256 hash function to build its immutable ledger capabilities, which is part of FIPS's Secure Hash Standard. See https://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.180-4.pdf
+- AWS Documentation on QLDB compliance: https://docs.aws.amazon.com/qldb/latest/developerguide/qldb-compliance.html
+- "[Data in transit is encrypted using TLS](https://docs.aws.amazon.com/qldb/latest/developerguide/data-protection.html)". AWS uses two implementations of TLS. Both have FIPS mode and non FIPS mode. It's not clear which is used or whether FIPS mode is on for QLDB
+    - OpenSSL: The open source standard implementation
+    - s2n: Amazon's own lightweight implementation
+- "[Data at rest is encrypted using AWS owned keys](https://docs.aws.amazon.com/qldb/latest/developerguide/data-protection.html)". There is no indication of what crypto is used by AWS for encryption.
+- QLDB is not in the list of services that have a FIPS compliant endpoints: https://aws.amazon.com/compliance/fips/?nc1=h_ls
+
+We can handwave around FIPS compliance for QLDB but for now it is not official, and we cannot know for sure how AWS's internal crypto is used.
```

Comment:
```suggestion
FIPS compliance for QLDB is unclear at this point mainly because we cannot know how AWS's internal crypto is used.
```

Reply:
I ended up implementing this change manually.